src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -129,21 +123,21 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                         fields.put(FieldName.TITLE, fields.get(FieldName.TITLE).replaceAll("\\s+", " ")); // Normalize whitespaces
                     } else if ("BT".equals(tag)) {
                         fields.put(FieldName.BOOKTITLE, value);
-                    } else if ("T2".equals(tag) && (fields.get(FieldName.JOURNAL) == null || "".equals(fields.get(FieldName.JOURNAL)))) {
+                    } else if (("T2".equals(tag) || "J2".equals(tag) || "JA".equals(tag)) && (fields.get(FieldName.JOURNAL) == null || "".equals(fields.get(FieldName.JOURNAL)))) {
```

Comment:
This would have been optimal to use a switch case, which is very efficient for strings, but that's a bit of personal taste ;)

Reply:
You are right in terms of efficiency, but I really don't like switches. It is just to easy to forget the `break` statement. That doesn't happen if you stick to `if` and always use braces.